doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4413,6 +4413,23 @@ The issues in this section are related to :ref:`nrfxlib:mpsl`.
 
 .. rst-class:: v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-0
 
+DRGN-25261: Radio events may stop running after a big temperature change triggers HFXO tuning.
+
+  **Affected platforms:** nRF54L15
+
+  **Workaround:** Increase the Kconfig option :kconfig:option:`CONFIG_MPSL_HFCLK_LATENCY` by 512 microseconds to allow tuning to complete.
```

Comment:
Should we give an absolute suggested value for CONFIG_MPSL_HFCLK_LATENCY like you did with the other known issue below?

Reply:
That would be board specific. Increasing it by 512 us allows tuning to run, given that the initially configured CONFIG_MPSL_HFCLK_LATENCY is high enough to start the HFXO